    public ConverterException(final Throwable cause) {
        super(cause);
    }

